bootstrap/apply_shop_updates.py

```
@@ -0,0 +1,60 @@
+def _run_shop_code(label: str, code: str) -> None:
+    python_exe = _venv_python(SHOP_BACKEND)
+    print(f"[apply_shop_updates] start: {label} (python={python_exe})", flush=True)
+    env = os.environ.copy()
+    pythonpath_parts = [str(ROOT)]
+    if existing := env.get("PYTHONPATH"):
+        pythonpath_parts.append(existing)
+    env["PYTHONPATH"] = os.pathsep.join(pythonpath_parts)
+    env["PYTHONIOENCODING"] = "utf-8"
+    env["PYTHONUTF8"] = "1"
+
+    result = subprocess.run([python_exe, "-c", code], cwd=str(SHOP_BACKEND), env=env)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)